### DISCUSSION TITLE
¿How to disable a room?

### DISCUSSION THREAD

**Discussion by Roberto-cpu-spec** [mozilla/hubs#3858] [Hubs Cloud Questions] [answered]
Hi community, can the room be disabled when the admin is not in the room? or can access to the room be blocked when the administrator is not in the room?

**Comment by robinkwilson** [comment, marked as answer]
That functionality isn't available yet in the core product (Though you **_could_** build this with a custom client with custom code, but I say that to everything :P). 

Here's the issue to watch: https://github.com/mozilla/hubs/issues/2486

**Reply by Roberto-cpu-spec** [reply]
Thanks, well I will learn code (in what programming language is it encoded?).
Regarding the costs of hubs cloud I have doubts (I know that there is no way to know an exact amount), I would like to know if the charge is for use and for the number of objects in the room? is there a way to set a payment limit?
I use hubs to teach classes, but I think that students can enter the room without the moderator being there and make use of it, generating high costs.

**Reply by Roberto-cpu-spec** [reply]
Hello again, I have received the update but I cannot make any changes in the main page, the only thing that has changed me is the colors of the letters of the page with the short domain. How can I change my company logo, background colors etc.? I leave some images of my page.
![Hubs1](https://user-images.githubusercontent.com/74120631/116013631-e35d1c00-a5f6-11eb-8e09-670bbbb09b1a.png)

![Hubs2](https://user-images.githubusercontent.com/74120631/116013595-9ed18080-a5f6-11eb-9ed8-4aa7212414ee.png)
